src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +58,38 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetIsTopLevelWrapper(debugID, isTopLevelWrapper) {
+    emitEvent('onSetIsTopLevelWrapper', debugID, isTopLevelWrapper);
   },
-  onMountComponent(internalInstance) {
-    emitEvent('onMountComponent', internalInstance);
+  onSetIsComposite(debugID, isComposite) {
```

Comment:
This is also an implementation detail. You can render a full React app with no native nodes and just using life-cycle methods. We could implement DOM as just a series of composites. Also, the notion of composite is overloaded because there are stateless function, functional and classes.

I don't think we need to expose this. Do you have use case?